### PR TITLE
Fix issue on interface not found

### LINKED ISSUE #88: Internal server error 500 on `/api/amlight/sdntrace_cp/traces`
I managed to find the root cause of the bug that @gretelliz and I were discussing [on this review/comment](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/222#pullrequestreview-1375386288). It turns out that it's a very specific bug related to a corner case that's been around for years in the code base. In summary, [`find_endpont(switch, port)`](https://github.com/kytos-ng/sdntrace_cp/blob/master/utils.py#L58-L59) when calling `switch.get_interface_by_port_no(port)` it's always assuming that the interface will be found, which in most cases will be true, but there's cases such as when OpenFlow port description hasn't been received yet and it's a new topology that hasn't been loaded from DB, then it can result in this issue. So, it needs to handle the case when the interface isn't found.  

For the record here's the traceback:

```
Apr  6 06:44:57 runner-rmtk68xb-project-107-concurrent-0 2023-04-06 06:44:57,827 (Thread-619) kytos.core.controller:ERROR app:1449:  Exception on /api/amlight/sdntrace_cp/traces [PUT]#012Traceback (most recent call last):#012  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 2073, in wsgi_app#012    response = self.full_dispatch_request()#012  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1519, in full_dispatch_request#012    rv = self.handle_user_exception(e)#012  File "/usr/local/lib/python3.9/dist-packages/flask_cors/extension.py", line 165, in wrapped_function#012    return cors_after_request(app.make_response(f(*args, **kwargs)))#012  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1517, in full_dispatch_request#012    rv = self.dispatch_request()#012  File "/usr/local/lib/python3.9/dist-packages/flask/app.py", line 1503, in dispatch_request#012    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)#012  File "/usr/local/lib/python3.9/dist-packages/kytos/core/helpers.py", line 402, in wrapper_validate#012    return func(*args, data=data, **kwargs)#012  File "//var/lib/kytos/napps/amlight/sdntrace_cp/main.py", line 82, in get_traces#012    results.append(self.tracepath(entry, stored_flows))#012  File "//var/lib/kytos/napps/amlight/sdntrace_cp/main.py", line 109, in tracepath#012    result = self.trace_step(switch, entries, stored_flows)#012  File "//var/lib/kytos/napps/amlight/sdntrace_cp/main.py", line 181, in trace_step#012    endpoint = find_endpoint(switch, port)#012  File "//var/lib/kytos/napps/../napps/amlight/sdntrace_cp/utils.py", line 59, in find_endpoint#012    if interface.link:#012AttributeError: 'NoneType' object has no attribute 'link'
```

@gretelliz can you add this to your radar for `2023.1` version? Thanks.

### PR DESCRIPTION
**Pull request by gretelliz**
Closes #88 

### Summary

When interface not found in a trace, it will be considered an `incomplete` trace. i.e. It is the same case as when t`race_step` early returns None if not flow or not port.

### Local Tests

[No warning seen.](https://github.com/kytos-ng/sdntrace_cp/blob/896fe9b9d7afff9aafcdf0d48ce5f395b193fd19/main.py#L183)

### End-to-End Tests

progress

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -179,6 +179,10 @@ def trace_step(self, switch, entries, stored_flows):
             return None
 
         endpoint = find_endpoint(switch, port)
+        if endpoint is None:
+            log.warning("Interface not found")
```
Comment (viniarck) [inline main.py:183]: It's nice to have this log.warning here, but can you augment this? 

I mean when this ever happens, as it is, there's no information about which endpoint exactly, so it won't be immediately evident. Maybe we can have something like `f"Port {port} not found on switch {switch}"`?

**Review by viniarck — CHANGES_REQUESTED** [review]
It's great to have this fixed, @gretelliz . Overall, looks good, asked minor changes though:

- Can you augment the log message? 
- Also, can you update the changelog?

File: main.py (hunk 1 above)
Comment (gretelliz) [reply main.py]: Thank you. Done!
